t/03components.t

```diff
@@ -72,8 +72,9 @@ use DateTime;
     is( $d->time,      '02:12:50', '->hms' );
     is( $d->time('!'), '02!12!50', q{->time('!')} );
 
-    is( $d->datetime, '2001-07-05T02:12:50', '->datetime' );
-    is( $d->iso8601,  '2001-07-05T02:12:50', '->iso8601' );
+    is( $d->datetime,       '2001-07-05T02:12:50', '->datetime' );
+    is( $d->datetime(q{ }), '2001-07-05 02:12:50', q{->datetime(q{ }} );
+    is( $d->iso8601,        '2001-07-05T02:12:50', '->iso8601' );
```

Comment:
Very very minor: it might be useful to add a test that if iso8601 is called with arguments, they're ignored.